lib/core/engine/iteration.js

```diff
@@ -9,6 +9,7 @@ const setResourceTimingBufferSize = require('../../support/setResourceTimingBuff
 const filterWhitelisted = require('../../support/userTiming').filterWhitelisted;
 const engineUtils = require('../../support/engineUtils');
 const Video = require('../../video/video');
+const TCPDUMP = require('../../support/tcpdump');
```

Comment:
Nitpick: We try to not overuse capital letters, so what about either TCPDump or TcpDump?